fdm-app/app/routes/farm.$b_id_farm.settings.tsx

```
@@ -0,0 +1,109 @@
+                farmOptions={loaderData.farmOptions}
+                b_id_farm={loaderData.b_id_farm}
+                action={undefined}
+            />
+            <main>
+                <FarmTitle
+                    title={"Instellingen"}
+                    description={"Beheer de instellingen van je bedrijf"}
+                />
+                <FarmContent
+                    sidebarItems={loaderData.sidebarPageItems}       
+                    children={<Outlet/>}             
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)